src/api/requestsApi.tsx

```diff
@@ -70,6 +70,11 @@ export interface RPARequest {
   linkedinPositionSummary?: string;
   linkedinQualifications: string[];
   dcwf: string[];
+  dcwf2: string[];
+  dcwf3: string[];
```

Comment:
Should these fields be converted to just string, since it is no longer an array of DCWFs, just single items?

Reply:
🤦